src/clock/index.html

```diff
@@ -0,0 +1,37 @@
+---
+title: Clock
+layout: base
+show_banner: False
+body_class: clock
+---
+
+<section>
+    <div class="center">
+        <div class="heading-container">
+            <h1 class="heading">Time is everything. Stay tuned.</h1>
+        </div>
+        <div class="countdown-container">
+            <div class="time-unit-container d-inline-block">
+                <div class="time-number">
+                    <span class="digits_container days-digits_container"></span><span>:</span><span class="digits_container hours-digits_container"></span><span>:</span><span class="digits_container minutes-digits_container"></span><span>:</span><span class="digits_container seconds-digits_container"><span>:</span></span></div>
+            </div>
+        </div>
+    </div>
+    <div class="animations">
+        {% asset clock/left_rectangle-large.svg class='svg-animated svg-animated--left svg-animated--large' %}
+        {% asset clock/right_rectangle-large.svg class='svg-animated svg-animated--right svg-animated--large' %}
+        {% asset clock/oval-large.svg class='svg-animated svg-animated--oval svg-animated--large' %}
+        {% asset clock/left_rectangle-medium.svg class='svg-animated svg-animated--left svg-animated--medium' %}
+        {% asset clock/right_rectangle-medium.svg class='svg-animated svg-animated--right svg-animated--medium' %}
+        {% asset clock/oval-medium.svg class='svg-animated svg-animated--oval svg-animated--medium' %}
+        {% asset clock/left_rectangle-small.svg class='svg-animated svg-animated--left svg-animated--small' %}
+        {% asset clock/right_rectangle-small.svg class='svg-animated svg-animated--right svg-animated--small' %}
+        {% asset clock/oval-small.svg class='svg-animated svg-animated--oval svg-animated--small' %}
```

Comment:
Taking into account that we could reduce some repeated files. I think it would be better to include all figures as background images.